tests/actors/build.rs

```diff
@@ -347,6 +365,8 @@ async fn main() -> anyhow::Result<()> {
             "messaging-receiver-smithy",
             Some(vec![caps::MESSAGING.into()]),
         ),
+        ("wrpc-pinger-component-preview2", None),
+        ("wrpc-ponger-component-preview2", None),
```

Comment:
NIT: Do we need the `preview2` here if there's no other version? IIRC `foobar-component-command` and `foobar-component-command-preview2` exist because there was another.